client/src/components/VideoCall.jsx

```diff
@@ -48,7 +50,61 @@ export function VideoCall({ roomUrl, record }) {
         console.debug("track-started", event);
       }
     });
-  };
+
+    callFrame.on("active-speaker-change", (event) => {
+      // console.log("active speaker change");
+      const speakerEvents = player.get("speakerEvents") || [];
```

Comment:
I think Nicolas has added an "append" method to empirica that would let us add a new entry to an existing list, saved on the player object. I don't know how it works, but could be useful in this case. We can look at the empirica code to see if we can figure that out. Is that something you're up for taking on, or something that we should do together next week?

Reply:
> I think Nicolas has added an "append" method to empirica that would let us add a new entry to an existing list, saved on the player object. I don't know how it works, but could be useful in this case. We can look at the empirica code to see if we can figure that out. Is that something you're up for taking on, or something that we should do together next week?

I've tried using append but it didn't work, so I used this instead. It will be nice if we can look at it together.